docs/application/native/guides/compliations/watchface-complication-provider.md

```diff
@@ -0,0 +1,247 @@
+# Watchface complication provider
+
+The watchface complication provider application is a service application to provide complication data.
+Complication provider application provides only data. The watchface application decides how to display the complications.
+The Complication Provider app provides the complication data by using the watchface complication provider api to the watchface app.
+ 
+# Prerequisites

```

Comment:
As sabeena21 commented, in the md file, "#" is used for the title of the page.
From next sub titles, please use "##" and "###".

```
## Prerequisites
```